### DISCUSSION TITLE
Documentation standardization surrounding DocumentRoot

### DISCUSSION THREAD

**Discussion by riversdev0** [firefly-iii/firefly-iii#6358] [General]
Hi -

I installed Firefly for the first time a week ago, so I'm brand new.  I noticed that there is some inconsistency within the [official documentation](https://docs.firefly-iii.org/) about the directory into which Firefly gets installed.  For example, some documentation pages will say `/var/www/firefly-iii/public` and others will say `/var/www/html/firefly-iii/public` (note the added `html`).  That was a challenge for me as a brand new, first-time user.  I'd like to help improve the documentation, but I don't know which of the two options is the one James prefers.

Furthermore, the self-hosted [Installation Guide](https://docs.firefly-iii.org/firefly-iii/installation/self_hosted/) mentions `/var/www` as the likely location for web files.  In the Linux distributions I'm familiar with, the path is `/var/www/html`.  Therefore, without modification to the Apache config, Firefly would never load from `/var/www/firefly-iii`.

Within the scope of self-hosting, is it possible to bring these multiple pieces of information into alignment?  I'd like to help, but I just need some guidance on the preferred alignment strategy.

Thanks!

**Comment by JC5** [comment]
Feel free to open a PR. Either way is fine with me. 👍
